admin/base/utils.py

```diff
@@ -97,8 +97,12 @@ def get_subject_rules(subjects_selected):
 
 
 def get_nodelicense_choices():
-    return NodeLicense.objects.values_list('id', 'name')
+    return NodeLicense.objects.exclude(license_id='OTHER').values_list('id', 'name')
```

Comment:
Is 'OTHER' some test license?  Is there a possibility that there will be more test licenses made in the future?  Main question is weather or not 'OTHER' should be hard-coded, or if we should make that a variable at the top that can be added to? If it's a one off, no need to change this!

Reply:
Hey @erinspace, thanks for the review.  My understanding, the "Other" option allows users to upload a license.txt file, but this isn't currently supported on the preprints workflow.  So I am just removing it as an option from the preprint provider portion of the admin app.  Other is still an option on projects.

<img width="843" alt="screen shot 2018-04-04 at 10 00 46 am" src="https://user-images.githubusercontent.com/9755598/38315945-2ffcd092-37ef-11e8-972c-81dc7e954954.png">